uefi/section.go

```diff
@@ -111,7 +153,7 @@ type Section struct {
 	fileOrder   int
 
 	// Type specific fields
-	TypeSpecific interface{} `json:",omitempty"`
+	TypeSpecific *TypeSpecificHeader `json:",omitempty"`
```

Comment:
I have a feeling it might be better to take the easy way out and just list all the type specific fields for all section types here with `json:",omitempty"` on each. Then the json will look nicer and we won't have to deal with marshaling interfaces.

Reply:
Maybe, lets explore this in another PR